### DISCUSSION TITLE
Tax ID

### DISCUSSION THREAD

**Discussion by tpazera** [vercel/community#4665] [Help] [answered]
### Summary

Hello,

I would like to upgrade my account to PRO, but on creating a new team I cannot set my company info (TAX ID) on payment tab.
My trail already expired and I deleted my old team after that. 

How can I provide info for the invoice before making a payment?

### Steps to Reproduce

1. Clicking Upgrade to PRO button
2. Setting team name
3. Card info required before going next

**Comment by amyegan** [comment, marked as answer]
Sorry you ran into this problem, @tpazera. We can't safely discuss account info here in public. Instead, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team for help with this billing issue. 🙏
